src/gar.lua

```diff
@@ -1054,7 +1054,7 @@ function gar.pruneGateways(currentTimestamp, msgId)
 				and garSettings ~= nil
 				and gateway.stats.failedConsecutiveEpochs >= garSettings.operators.failedEpochCountMax
 			then
-				-- slash 20% of the minimum operator stake and return the rest to the protocol balance, then mark the gateway as leaving
+				-- slash the minimum operator stake and return it to the protocol balance, mark the gateway as leaving which will vault and remaining stake
```

Comment:
nit: `balance. Mark the gateway as leaving which will vault any remaining stake`

Punctuation and typo.